tests/providers/amazon/aws/operators/test_sagemaker_base.py

```diff
@@ -40,3 +45,19 @@ def test_default_integer_fields(self):
         self.sagemaker.preprocess_config()
 
         assert self.sagemaker.integer_fields == EXPECTED_INTEGER_FIELDS
+
+
+class TestSageMakerExperimentOperator:
+    @patch("airflow.providers.amazon.aws.hooks.sagemaker.SageMakerHook.conn", new_callable=mock.PropertyMock)
+    def test_create_experiment(self, conn_mock):
+        conn_mock().create_experiment.return_value = {"ExperimentArn": "abcdef"}
+
+        op = SageMakerCreateExperimentOperator(
+            name="the name", description="the desc", tags={"the": "tags"}, task_id="whatever"
```

Comment:
```suggestion
            name="the name", description="the desc", tags={"the": "{{ task.task_id }}"}, task_id="whatever"
```

Reply:
is this supposed to be replaced by the task ID ? It doesn't work for me when I run it locally (even after fixing the transform to happen in the execute rather than in the ctor)...